### ISSUE TITLE
options.before/after in insertIntoFile only supports valid regex

### ISSUE THREAD

**Issue by adamjmcgrath** [ember-cli/ember-cli#7473]
The `insertIntoFile` example in the docs (https://ember-cli.com/api/classes/Blueprint.html#method_insertIntoFile) doesn't work in ember-cli@2.17+ because of the regex feature added to `options.before/after` in #7384

```js
''.match('Router.map(function() {')
// SyntaxError: Invalid regular expression: /Router.map(function() {/: Unterminated group
```

I noticed it when trying to install ember-cli-mirage, because of https://github.com/samselikoff/ember-cli-mirage/blob/master/blueprints/ember-cli-mirage/index.js#L48

I'll probably raise a PR to workaround it in `ember-cli-mirage` unless you think it's something that should be addressed in `ember-cli`?

**Comment by kellyselden** [comment]
This looks like something I broke. I will look into it ASAP.

**Comment by kellyselden** [comment]
I see the issue, working on a fix.

**Comment by kellyselden** [comment]
Should be fixed in #7475
